ReactiveCocoa/Swift/Signal.swift

```diff
@@ -128,8 +128,8 @@ public final class Signal<Value, Error: ErrorType> {
 		}
 
 		if let token = token {
-			return ActionDisposable {
-				self.atomicObservers.modify { observers in
+			return ActionDisposable { [weak self] in
```

Comment:
Curious: do tests pass with this as `unowned`? I'd rather do that since it defines the relationship more correctly.